android/demos/compose/src/main/kotlin/com/uber/rib/compose/root/RootView.kt

```diff
@@ -33,7 +35,7 @@ class RootView @JvmOverloads constructor(
       TextView(context).apply {
         text = "root (view)"
         setTextColor(Color.WHITE)
-      }
+      },
```

Comment:
I don't get this trailing comma on here. (I do get and support the trailing comma on constructor). Is this auto by `googleStyle`?

Reply:
That's auto by `ktlint` since [Trailing comma on call site](https://pinterest.github.io/ktlint/rules/standard/#trailing-comma-on-call-site) became one of standard rules